fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/table/GenericTableLoader.java

```diff
@@ -83,7 +87,7 @@ private void configureTables(
                     new DataSourceMetadata(
                             dataSourceConfiguration.getName(),
                             Collections.emptyMap(),
-                            dataSourceConfiguration.getDataSegmentMetadata()
+                            dataSourceConfiguration.getDataSegments()
```

Comment:
This is not a new method. It's the old method that's been renamed. The reason is mentioned in previous comments.